modules/update-vsphere-virtual-hardware-on-template.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// updating/updating-hardware-on-nodes-running-in-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="update-vsphere-virtual-hardware-on-template_{context}"]
+= Updating the virtual hardware for template on vSphere
+
+.Prerequisites
+
+* You have cluster administrator permissions to execute the required permissions in the vCenter instance hosting your {product-title} cluster.
+* Your vSphere ESXi hosts are version 6.7U3 or later.
+
+.Procedure
+
+. If the RHCOS template is configured as a vSphere template follow link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-D632CAC5-BA5E-4A1E-959B-382D9ACB1DD0_copy.html[Convert a Template to a Virtual Machine]
+in the VMware documentation prior to the next step.
+. Update the VM in the vSphere client. Follow link:https://kb.vmware.com/s/article/1010675[Upgrading a virtual machine to the latest hardware version] in the VMware documentation for more information.
```

Comment:
You mentioned needing to ensure that the vm is never started. Should that be called out?

Reply:
yeah, i think that would be a good point to call out.  it is easy to accidentally ignite the template.  otherwise, LGTM

Reply:
all set